scripts/server_benchmark.py

```diff
@@ -11,17 +11,20 @@
 import argparse
 import cProfile
 
-import ga4gh.backend as backend
-import ga4gh.protocol as protocol
-import ga4gh.datarepo as datarepo
-
 import guppy
 
+import utils
+utils.ga4ghImportGlue()
+import ga4gh.backend as backend  # noqa
+import ga4gh.protocol as protocol  # noqa
+import ga4gh.datarepo as datarepo  # noqa
+
 
 class HeapProfilerBackend(backend.Backend):
-    def __init__(self, dataDir):
-        dataRepository = datarepo.FileSystemDataRepository(dataDir)
-        super(HeapProfilerBackend, self).__init__(dataRepository)
+    def __init__(self, registryDb):
+        repo = datarepo.SqlDataRepository(registryDb)
+        repo.open(datarepo.MODE_READ)
+        super(HeapProfilerBackend, self).__init__(repo)
```

Comment:
I assume this is general cleanup, not immediately related to the `maxResponseLength` -> `maxBufferSize` issue?